ivy/functional/ivy/general.py

```diff
@@ -2177,13 +2177,18 @@ def stable_pow(
 
 
 @handle_exceptions
-def get_all_arrays_in_memory():
+def get_all_arrays_in_memory() -> ivy.Array:
```

Comment:
This function returns a list of arrays
```suggestion
def get_all_arrays_in_memory() -> List[ivy.Array, ivy.NativeArray]:
```

Reply:
My bad
 i will fix the code and push it tomorrow